main.go

```diff
@@ -67,14 +70,30 @@ func init() {
 	// +kubebuilder:scaffold:scheme
 }
 
+func runPrometheusServer(metricsAddr string) {
+	setupLog.Info("Entered runPrometheusServer")
+	mux := restful.NewContainer()
+	mux.Handle("/metrics", promhttp.Handler())
+	setupLog.Info("Defining prom server")
```

Comment:
maybe write Prometheus in full as the admin will see those logs

Reply:
you are right